tests/letstest/multitester.py

```diff
@@ -63,10 +63,8 @@
 parser.add_argument('test_script',
                     default='test_letsencrypt_auto_certonly_standalone.sh',
                     help='path of bash script in to deploy and run')
-#parser.add_argument('--script_args',
-#                    nargs='+',
-#                    help='space-delimited list of arguments to pass to the bash test script',
-#                    required=False)
+parser.add_argument('--path',
```

Comment:
I kept the optimizations, and did the call with `--repo` alone. I use `{toxinidir}` instead of `{env:SYSTEM_DEFAULTWORKINGDIRECTORY}` to make these toxenvs less coupled with Azure. I also added a `AWS_C2_PEM_FILE` environment variable consumed by these toxenvs to make them totally decoupled from Azure.